packages/webapp/queries/base/requests/proposalsQuery.ts

```diff
@@ -15,16 +15,14 @@ export interface ProposalsResponse {
 export const getProposals = async (
   chainId: number,
   token: string,
-  limit: number = 100,
-  page?: number
+  limit: number = 200,
```

Comment:
any reason this is being changes and not just used like `getProposals(chainId, token, 200)`? or even just updating the default value for limit and calling `getProposals(chainId, token)`?